lib/data/beacon_region.dart

```diff
@@ -44,7 +44,7 @@ class BeaconRegionIBeacon extends BeaconRegion {
 
   String get proximityUUID => ids[0];
 
-  int get major => ids.length > 1 ? ids[1] : null;
+  String get major => ids.length > 1 ? ids[1] : null;
```

Comment:
Tested this one on his example and it doesn't detect any beacons.
You're supposed to convert the Major and Minor to int or decimal.
changing it to string will convert the ids to string.